### PR TITLE
Task/dogfooding

### PR DESCRIPTION
**Pull request by filbabic**

### CODE REVIEW

File: dogfooding/src/main/java/io/getstream/chat/android/dogfooding/DogfoodingApp.kt [hunk 1]
```diff
@@ -35,16 +48,25 @@ class DogfoodingApp : Application() {
...
             loggingLevel = loggingLevel
         ).build().also {
-            streamCalls = it
+            calls = it
         }
     }
+
+    fun logOut() {
+        FirebaseAuth.getInstance().signOut()
+        credentialsProvider
```
Comment (skydoves) [inline dogfooding/src/main/java/io/getstream/chat/android/dogfooding/DogfoodingApp.kt]: Seems to added by mistake?

File: dogfooding/src/main/java/io/getstream/chat/android/dogfooding/UserPreferencesImpl.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        val editor = sharedPreferences.edit()
+
+        editor.putString(KEY_ID, userCredentials.id)
+        editor.putString(KEY_TOKEN, userCredentials.token)
+        editor.putString(KEY_IMAGE, userCredentials.image)
+        editor.putString(KEY_NAME, userCredentials.name)
+        editor.putString(KEY_ROLE, userCredentials.role)
+
+        editor.apply()
+    }
```
Comment (skydoves) [inline dogfooding/src/main/java/io/getstream/chat/android/dogfooding/UserPreferencesImpl.kt]: We can refactor like this with an `edit` extension.

```kotlin
sharedPreferences.edit {
            putString(KEY_ID, userCredentials.id)
            putString(KEY_TOKEN, userCredentials.token)
            putString(KEY_IMAGE, userCredentials.image)
            putString(KEY_NAME, userCredentials.name)
            putString(KEY_ROLE, userCredentials.role)   
}
```

Comment (filbabic) [reply dogfooding/src/main/java/io/getstream/chat/android/dogfooding/UserPreferencesImpl.kt]: yeah that's true - good call! :D

File: dogfooding/src/main/java/io/getstream/chat/android/dogfooding/DogfoodingApp.kt (hunk 1 above)
Comment (filbabic) [reply dogfooding/src/main/java/io/getstream/chat/android/dogfooding/DogfoodingApp.kt]: Yeah it should be clearing instead! thanks!

**Pull request merged**